packages/components/src/components/link-tile/link-tile-styles.ts

```diff
@@ -91,19 +96,21 @@ export const getComponentCss = (
       }),
       p: {
         color: getThemedTypographyColor('dark', 'primary'),
-        ...textSmallStyle,
-        maxWidth: pxToRemWithUnit(550),
+        maxWidth: pxToRemWithUnit(550), // in this case rem unit makes sense to scale up available space
         margin: 0,
+        ...textLargeStyle,
         ...mergeDeep(
           buildResponsiveStyles(size, (s: LinkTileSize) => sizeMap[s]),
           buildResponsiveStyles(weight, (w: LinkTileWeight) => ({ fontWeight: getFontWeight(w) }))
         ),
       },
     },
     root: {
+      display: 'block',
```

Comment:
Probably unneeded.

Reply:
Yeah, true. And what about `height: 'fit-content',` ? Do you know why it was implemented?

Reply:
No, but seems useless, too.

Reply:
I think you added/removed the wrong `display: 'block'`.

Reply:
True. Done.